### ISSUE TITLE
Tabbed fieldset are all hidden by default

### ISSUE THREAD

**Issue by bkfox** [matthiask/django-content-editor#13]
Provided the following admin class:

```python
class PageAdmin(ContentEditor, TreeAdmin):
    list_display = ["indented_title", "move_column", "is_active"]
    prepopulated_fields = {"slug": ("title",)}
    raw_id_fields = ["parent"]

    fieldsets = (
        ('Main', {'fields': ('title', 'slug', 'by'),
                  'classes': ('tabbed', 'active')}),
        ('Settings', {'fields': ('allow_comments', 'featured'),
                      'classes': ('tabbed',)}),
    )

    inlines = [
        plugins.richtext.RichTextInline.create(models.RichText),
        plugins.image.ImageInline.create(models.Image),
    ]
```
All tabs are closed at page load; after a quick look at `tabbed_fieldsets.js`, it seems there is no way to have one opened by default, which would be a really cool feature IMHO. So here is a feature request: the first fieldset with both 'tabbed' and 'active' classes will be displayed to user instead of being hidden.

**Comment by matthiask** [comment]
Thanks for the suggestion! Released as 1.4.2 (with `uncollapse` instead of `active`)

**Comment by bkfox** [comment]
Thanks! However, I see an oversight in the commit, at line 55 ```if (errorIndex || uncollapseIndex)``` should be ```if (errorIndex === null || uncollapseIndex === null)```, since uncollapseIndex might be 0.

**Comment by matthiask** [comment]
Oh right, now I feel stupid :) fixed and released as 1.4.3.
